messages/teleporter/message.go

```diff
@@ -35,7 +35,7 @@ type ReceiveCrossChainMessageInput struct {
 }
 
 // unpack Teleporter message bytes according to EVM ABI encoding rules
-func unpackTeleporterMessage(messageBytes []byte) (*TeleporterMessage, error) {
+func UnpackTeleporterMessage(messageBytes []byte) (*TeleporterMessage, error) {
```

Comment:
Does this need to be exported? I'm not seeing any references to it outside `package teleporter`.

Reply:
this is used in `e2e_test.go`